docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>There is a broken sentence here:</p><p> </p><blockquote>The matrix&nbsp;A&nbsp;is also determined by the of product formula to use&nbsp;</blockquote><p> </p><p>I think it might be good to emphasize here, that we are using time-independent (i.e. static) coefficients. In particular, using <code>symmetric=True</code> means that we are using the definition for <code>A</code> as given by <a href="https://quantum-journal.org/papers/q-2023-07-25-1067" target="_blank">https://quantum-journal.org/papers/q-2023-07-25-1067/</a> which could be pointed out explicitly (or one could at least link to the API docs). This stands out, because every other argument is explicitly called out in the text. Alternatively, you can simply leave this be the default value (which would be <code>False</code> though, so it would be good to point out that you are overwriting the default here if you want to keep it).   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>